deblur/parallel_deblur.py

```diff
@@ -0,0 +1,160 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+from __future__ import division
+import multiprocessing as mp
+import traceback
+import sys
+import subprocess
+from functools import partial
+import logging
+
+
+def deblur_system_call(params, input_fp):
+    """Build deblur command for subprocess.
+
+    Parameters
+    ----------
+    params: list of str
+        parameter settings to pass to deblur CLI
+    input_fp : str
+        name of the input fasta file to deblur
+
+    Returns
+    -------
+    stdout: string
+        process output directed to standard output
+    stderr: string
+        process output directed to standard error
+    return_value: integer
+        return code from process
+
+    """
+    logger = logging.getLogger(__name__)
+    logger.debug('deblur system call params %s, input_fp %s' %
+                 (params, input_fp))
+
+    # construct command
+    script_name = "deblur"
+    script_subprogram = "workflow"
+    command = [script_name,
+               script_subprogram,
+               '--seqs-fp', input_fp,
+               '--is-worker-thread',
+               '--keep-tmp-files']
+    command.extend(params)
+
+    logger.debug('running command %s' % command)
+    return system_call(command)
+
+
+def system_call(cmd):
```

Comment:
There is already a `system_call` function in workflow.py: https://github.com/amnona/deblur/blob/23b55ba9671dd2f5f2050fe7764cbbfc71ab9e4a/deblur/workflow.py#L642

can you import it from there to avoid code duplication?